### ISSUE TITLE
Wrong indentation in output

### ISSUE THREAD

**Issue by 90yukke** [polishgeeks/polishgeeks-dev-tools#42]
```
...
Rspec files names: 24 files checked      OK
Rspec files structure (25 checked)       OK
Tasks files names: 0 files checked       OK
Rspec (151 ex, 0 fa, 0 pe)               OK
SimpleCov covered 100.0%, required 97.0%  OK
Yard                                     OK
ExamplesComparator                       OK
...
```

**Comment by nijikon** [comment]
@90yukke what version are you using?

**Comment by 90yukke** [comment]
@nijikon `polishgeeks-dev-tools (1.2.1)`

**Comment by nijikon** [comment]
Ok, this is already fixed on master. Will close when new version is released.

**Comment by nijikon** [comment]
Fixed in 1.3.0.
